bin/steps/nltk

```diff
@@ -21,17 +20,30 @@ EXPORT_PATH="${BUILDPACK_DIR}/export"
 if is_module_available 'nltk'; then
 	output::step "Downloading NLTK corpora..."
 
-	nltk_packages_definition="$BUILD_DIR/nltk.txt"
+	nltk_packages_definition="${BUILD_DIR}/nltk.txt"
 
-	if [[ -f "$nltk_packages_definition" ]]; then
+	if [[ -f "${nltk_packages_definition}" ]]; then
 		meta_set "nltk_downloader" "enabled"
 
-		readarray -t nltk_packages <"$nltk_packages_definition"
+		readarray -t nltk_packages <"${nltk_packages_definition}"
 		output::step "Downloading NLTK packages: ${nltk_packages[*]}"
 
-		python -m nltk.downloader -d "$BUILD_DIR/.heroku/python/nltk_data" "${nltk_packages[@]}" | output::indent
-		set_env NLTK_DATA "/app/.heroku/python/nltk_data"
+		nltk_data_dir="/app/.heroku/python/nltk_data"
 
+		if ! python -m nltk.downloader -d "${nltk_data_dir}" "${nltk_packages[@]}" |& output::indent; then
```

Comment:
@edmorley the previous code downloaded to `$BUILD_DIR/.heroku/python/nltk_data` but the new code downloads to `/app/.heroku/python/nltk_data`. Won't that be a problem?

Reply:
No since `/app/.heroku/python` is a symlink to the equivalent build location (it just results in a nicer message in teh build log, as seen in the test fixture update) :-)